targets/ipq40xx-mikrotik

```diff
@@ -0,0 +1,6 @@
+include 'mikrotik.inc'
+
+device('mikrotik-sxtsq-5-ac', 'mikrotik_sxtsq-5-ac', {
+	factory = false,
+	aliases = {'mikrotik-discg-5acd'},
```

Comment:
Im actualy using a MikroTik RBDiscG-5acD not a RBSXTsqG-5acD but in openwrt there is no seperate build for that device since the hardware is identical.
![image](https://user-images.githubusercontent.com/943976/177373608-85d06d20-f3c9-4471-8bec-50e9b288dc96.png)